nix/scripts.nix

```diff
@@ -7,6 +7,7 @@ let
       services.cardano-exporter = {
         enable = true;
         inherit (envConfig) genesisHash;
+        inherit (envConfig) genesisFile;
```

Comment:
this can also be done as `inherit (envConfig) genesisHash genesisFile);`

Reply:
Fixed. Thanks.